logproof/tests/seal.rs

```diff
@@ -0,0 +1,293 @@
+use ark_ff::{Fp, MontBackend, MontConfig};
+use ark_poly::univariate::DensePolynomial;
+use merlin::Transcript;
+use seal_fhe::{
+    BFVEncoder, BfvEncryptionParametersBuilder, CoefficientModulus, Context, Decryptor, Encryptor,
+    KeyGenerator, PlainModulus, PolynomialArray, SecurityLevel,
+};
+
+use logproof::{
+    fields::{FpRistretto, SealQ128_1024, SealQ128_2048, SealQ128_4096, SealQ128_8192},
+    linear_algebra::Matrix,
+    math::{make_poly, next_higher_power_of_two, Rem, Zero},
+    test::{
+        bfv_delta, convert_to_polynomial, convert_to_smallint, strip_trailing_value, LatticeProblem,
+    },
+    InnerProductVerifierKnowledge, LogProof, LogProofGenerators, LogProofProverKnowledge,
+    LogProofTranscript,
+};
+
+fn full_knowledge_proof<F, const N: usize>(
```

Comment:
I think we should call this "test_seal_linear_relation"

Reply:
Sounds good! Changed